src/Parser/Reviews/ReviewerParser.php

```diff
@@ -56,6 +55,8 @@ public function getUrl(): string
         if ($node->count()) {
             return $node->attr('href');
         }
+
+        return "";
```

Comment:
@pushrbx This is indeed the appropriate paradigm here. Actually, I chose not to return anything as a fallback here because URLs parsed from MAL are always supposed return something and never empty string, null or any other data type. So it would throw an exception if nothing is matched - alerting us that the parser for getting the URL here broke (due to HTML change).

Reply:
That throwing of exception should be done explicitly here, because that way anyone reads the code will immidiately understand that this bit always should return something. I'll add something here according to this idea.

Reply:
Sounds good